lib/SelfRepairInteraction.js

```diff
@@ -21,4 +21,8 @@ exports.SelfRepairInteraction = {
   setSelfRepair(enabled) {
     Preferences.set('browser.selfsupport.enabled', enabled);
   },
+
+  isEnabled() {
+    return Preferences.get('browser.selfsupport.enabled', true);
```

Comment:
I'm assuming this works like 'get and return the value if it has been set (which may return `false`), otherwise return the default value of `true`?'